go2.lic

```diff
@@ -183,7 +183,27 @@ if XMLData.game =~ /^DR/
 
   # Iterate through the aggregated map wayto overrides from above and set new stringprocs
   wayto_overrides.each do | key, values |
-    Map.list[values['start_room'].to_i].wayto["#{values['end_room']}"] = StringProc.new("#{values['str_proc']}")
+    start_room_id = values['start_room'].to_i
+    end_room_id = values['end_room'].to_i
+
+    start_room = Map.list[start_room_id]
+
+    old_wayto = start_room.wayto["#{end_room_id}"]
+    old_timeto = start_room.timeto["#{end_room_id}"]
+
+    new_wayto = old_wayto
+    new_timeto = old_timeto
+
+    if values['str_proc']
+      new_wayto = StringProc.new("#{values['str_proc']}")
+    end
+
+    if values['travel_time']
+      new_timeto = values['travel_time'].to_f
+    end
```

Comment:
Conditionally override the `wayto` and `timeto`. This lets you make a path less desirably by increasing the `timeto` without necessarily needing to override the way in which you move between the two rooms, and vice versa. Or, you can override both!

```yaml
personal_wayto_overrides:
  test:
    start_room: 12245
    end_room: 12240
    travel_time: 0.1
    str_proc: move ne; move sw; move e
```
```yaml
personal_wayto_overrides:
  test:
    start_room: 12245
    end_room: 12240
    travel_time: 50
```
```yaml
personal_wayto_overrides:
  test:
    start_room: 12245
    end_room: 12240
    str_proc: move ne; move sw; move e
```